SEC/Equities/EquitiesExampleIndividuals.rdf

```diff
@@ -95,6 +274,134 @@
+		<fibo-fnd-rel-rel:hasTag>BBG001S8CRC3</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;BBG009S39JY5">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;FinancialInstrumentGlobalIdentifier"/>
+		<rdfs:label>BBG009S39JY5</rdfs:label>
+		<skos:definition>Alphabet Inc. class A share class Financial Instrument Global Identifier (FIGI)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;AlphabetIncClassACommonStock"/>
+		<fibo-fnd-rel-rel:hasTag>BBG009S39JY5</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;BBG009S3NB21">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;FinancialInstrumentGlobalIdentifier"/>
+		<rdfs:label>BBG009S3NB21</rdfs:label>
+		<skos:definition>Alphabet Inc. class C share class Financial Instrument Global Identifier (FIGI)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;AlphabetIncClassCCapitalStock"/>
+		<fibo-fnd-rel-rel:hasTag>BBG009S3NB21</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;BBG009S4MT03">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;FinancialInstrumentGlobalIdentifier"/>
+		<rdfs:label>BBG009S4MT03</rdfs:label>
+		<skos:definition>Alphabet Inc. class A share class Financial Instrument Global Identifier (FIGI) listed in the Nasdaq (NASDAQ/NGS)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;XNASListedAlphabetIncClassACommonStock"/>
+		<fibo-fnd-rel-rel:hasTag>BBG009S4MT03</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;BBG009S4MVF2">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;FinancialInstrumentGlobalIdentifier"/>
+		<rdfs:label>BBG009S4MVF2</rdfs:label>
+		<skos:definition>Alphabet Inc. class C share class Financial Instrument Global Identifier (FIGI) listed in the Nasdaq (NASDAQ/NGS)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;XNASListedAlphabetIncClassCCapitalStock"/>
+		<fibo-fnd-rel-rel:hasTag>BBG009S4MVF2</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;BBG00JPR0LX8">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;FinancialInstrumentGlobalIdentifier"/>
+		<rdfs:label>BBG00JPR0LX8</rdfs:label>
+		<skos:definition>Apple Inc. share class Financial Instrument Global Identifier (FIGI) listed in the London Stock Exchange</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;XLOMListedAppleIncCommonStock"/>
+		<fibo-fnd-rel-rel:hasTag>BBG00JPR0LX8</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;CUSIP-02079K107">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;CommitteeOnUniformSecuritiesIdentificationProceduresNumber"/>
```

Comment:
We should probably review this URI for the CUSIP class

Reply:
Raise an issue - I modeled these following what we had done for other generated identifiers.

Reply:
Raised #867